sky/job/controller.py

```diff
@@ -370,27 +373,27 @@ def run(self):
             # affect the jobs in terminal states.
             # We need to call set_cancelling before set_cancelled to make sure
             # the table entries are correctly set.
-            callback_func = spot_utils.event_callback_func(
+            callback_func = job_utils.event_callback_func(
                 job_id=self._job_id,
                 task_id=task_id,
                 task=self._dag.tasks[task_id])
-            spot_state.set_cancelling(job_id=self._job_id,
-                                      callback_func=callback_func)
-            spot_state.set_cancelled(job_id=self._job_id,
+            job_state.set_cancelling(job_id=self._job_id,
                                      callback_func=callback_func)
+            job_state.set_cancelled(job_id=self._job_id,
+                                    callback_func=callback_func)
 
 
 def _run_controller(job_id: int, dag_yaml: str, retry_until_up: bool):
```

Comment:
```suggestion
def _run_controller(controller_job_id: int, dag_yaml: str, retry_until_up: bool):
```
should we rename it to reduce the confusion w/ managed job id?

Reply:
SG!